detect_mechanical_parts/DetectionMsg[] detections